packages/main/src/ColorPicker.js

```diff
@@ -173,6 +175,72 @@ class ColorPicker extends UI5Element {
 		this.style.setProperty("--ui5_Color_Picker_Progress_Container_Color", this.color);
 	}
 
+	async onAfterRendering() {
+		if (isIE()) {
+			await renderFinished();
+			this._applySliderStyles();
+		}
+	}
+
+	_applySliderStyles() {
+		const hueSlider = this.getDomRef().querySelector(".ui5-color-picker-hue-slider").shadowRoot,
+			alphaSlider = this.getDomRef().querySelector(".ui5-color-picker-alpha-slider").shadowRoot;
+
+		if (hueSlider.children.length === 0 || alphaSlider.children.length === 0) {
+			return;
+		}
+
+		// ui5-slider::part(slider-handle)
+		hueSlider.querySelector(".ui5-slider-handle").style.width = "11px";
+		hueSlider.querySelector(".ui5-slider-handle").style.height = "1.25rem";
+		hueSlider.querySelector(".ui5-slider-handle").style.background = "transparent";
+		hueSlider.querySelector(".ui5-slider-handle").style.marginLeft = "-2px";
+		hueSlider.querySelector(".ui5-slider-handle").style.marginTop = "1px";
+
+		alphaSlider.querySelector(".ui5-slider-handle").style.width = "11px";
+		alphaSlider.querySelector(".ui5-slider-handle").style.height = "1.25rem";
+		alphaSlider.querySelector(".ui5-slider-handle").style.background = "transparent";
+		alphaSlider.querySelector(".ui5-slider-handle").style.marginLeft = "-2px";
+		alphaSlider.querySelector(".ui5-slider-handle").style.marginTop = "1px";
+
+		// ui5-slider::part(slider-handle)::after
+		// Skipped because it is pseudo element
+
+		// ui5-slider::part(progress-container)
+		hueSlider.querySelector(".ui5-slider-progress-container").style.width = "calc(100% + 11px)";
```

Comment:
Maybe get these just ones as constants to avoid some repetition:

hueSlider.querySelector(".ui5-slider-progress-container")
hueSlider.querySelector(".ui5-slider-handle")
alphaSlider.querySelector(".ui5-slider-progress-container")
alphaSlider.querySelector(".ui5-slider-handle")